### PR TITLE
test:add container resize and restart test

### PR DESCRIPTION
**Pull request by Dewey-Ding**
Signed-off-by: Dewey-Ding <deweyding@gmail.com>

### Ⅰ. Describe what this PR did
add container resize and restart test

### CODE REVIEW

**Comment by allencloud** [comment]
Could you help to review this? @ZouRui89 
Thanks a lot.

File: client/container_resize_test.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		t.Fatalf("expected a Server Error, got %v", err)
+	}
+}
+
+func TestContainerResize(t *testing.T) {
+	expectedURL := "/containers/container_id/resize"
+
+	httpClient := newMockClient(func(req *http.Request) (*http.Response, error) {
+		if !strings.HasPrefix(req.URL.Path, expectedURL) {
+			return nil, fmt.Errorf("Expected URL '%s', got '%s'", expectedURL, req.URL)
```
Comment (zhuangqh) [inline client/container_resize_test.go]: According to the [code style rules of pouch](https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md), No matter log or error, first letter of the message must be lower-case

Comment (Dewey-Ding) [reply client/container_resize_test.go]: I am following the example write by allencloud in issue #966 .Is there special?

Comment (ZouRui89) [reply client/container_resize_test.go]: I am afraid that @zhuangqh has made a point and we should change all the case problem in mock test. And I have noticed that the same problem exists in other part of Pouch. WDYT? @allencloud

Comment (allencloud) [reply client/container_resize_test.go]: Yes, I think what @zhuangqh mentioned is reasonable. Could someone fix this totally in a single pull request. @ZouRui89

Comment (ZouRui89) [reply client/container_resize_test.go]: @Dewey-Ding Could you fix the case problem? Thx a lot.

Comment (Dewey-Ding) [reply client/container_resize_test.go]: maybe i can do it in this pr.

Comment (Dewey-Ding) [reply client/container_resize_test.go]: @allencloud @ZouRui89

**Comment by ZouRui89** [comment]
LGTM

**Pull request merged**